### ISSUE TITLE
batch normalization 분산비교하는 내용에 대한 질문입니다.

### ISSUE THREAD

**Issue by hwanginsun** [public-ai/ALAI_Homework#86]
batch normalization을 사용한 경우와, 사용하지 않은 경우에 대해서 확인하려고 합니다

현재 작성중인 코드입니다

github page : https://github.com/hwanginsun/Batch_normalization

script : https://github.com/hwanginsun/Batch_normalization/blob/master/Batch_normalization_test.ipynb
